Bottom: dc22adfbae0b11548908dcff0e8662d0b9d7cf9b
Top:    18e33445dc6f810c7cba16be1239682e004ca642
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-14 01:51:42 +0100

drc: HACK HACK HACK


---

diff --git a/src/find.c b/src/find.c
index eb4cac2..73468d2 100644
--- a/src/find.c
+++ b/src/find.c
@@ -2748,7 +2748,12 @@ IsPolygonInPolygon (PolygonTypePtr P1, PolygonTypePtr P2)
 
               line.Point1.X = v->point[0];
               line.Point1.Y = v->point[1];
-              line.Thickness = 2 * Bloat;
+              line.Thickness = Bloat;
+              /* Another Bloat is added by IsLineInPolygon, making the correct
+               * 2x Bloat. Ideally we would change it there, but doing so
+               * breaks some other DRC checks which rely on the broken behaviour
+               * in IsLineInPolygon.
+               */
               line.Clearance = 0;
               line.Flags = NoFlags ();
               for (v = v->next; v != &c->head; v = v->next)
